src/EnergyPlus/HVACManager.cc

```diff
@@ -2951,63 +2803,59 @@ void SetHeatToReturnAirFlag(EnergyPlusData &state)
     // METHODOLOGY EMPLOYED:
     // Uses program data structures AirLoopControlInfo and ZoneEquipInfo
 
-    // Using/Aliasing
-    int NumPrimaryAirSys = state.dataHVACGlobal->NumPrimaryAirSys;
-    using ScheduleManager::CheckScheduleValue;
-    using ScheduleManager::GetCurrentScheduleValue;
-    using ScheduleManager::GetScheduleMaxValue;
-
-    bool CyclingFan(false); // TRUE means air loop operates in cycling fan mode at some point
```

Comment:
[src/EnergyPlus/HVACManager.cc:2960]:(style),[variableScope],The scope of the variable 'CyclingFan' can be reduced.